x-pack/libbeat/common/cloudfoundry/dopplerconsumer.go

```diff
@@ -80,23 +81,49 @@ func (c *DopplerConsumer) Run() {
 }
 
 func (c *DopplerConsumer) logsFirehose() {
-	c.firehose(c.callbacks.Log, consumer.LogMessages)
+	filter := FirehoseFilterAllLogs
```

Comment:
If we are sure that `FirehoseFilterAllLogs` and `FirehoseFilterLogs` collect the same logs, but `FirehoseFilterLogs` is much more efficient, wdyt about using `FirehoseFilterLogs` here and don't expose the option to the user?
```suggestion
	filter := FirehoseFilterLogs
```